source/mainnet/net/resources/glossary.rst

```diff
@@ -199,6 +199,13 @@ Cool-down period
 
 A period of time during which a transaction is frozen. Examples of when cool-down periods apply include removing a baker and updating stake. The length of a cool-down period varies between transactions.
 
+.. _glossary-concordium-bft-protocol:
+
+Concordium Byzantine Fault Tolerance (BFT) protocol
+===================================================
+
+The consensus protocol for the blockchain. The protocol offers higher transaction throughput and lower confirmation time because a block can be produced as soon as the previous block has been signed without waiting for a slot with an election winner. The protocol proceeds by rounds. In each round, a predetermined leader among the bakers should produce a block. The members of the finalization committee then sign this block, and their collective signatures are aggregated to form a quorum certificate. This quorum certificate is then included in the next block. If the leader fails to produce a block in the round, or not enough signatures were gathered for a QC, then the finalizers will instead send timeout messages, which are aggregated to form a timeout certificate. Each block either contains a quorum certificate or a timeout certificate for the previous round. When blocks on a common chain in two consecutive rounds have quorum certificates, the block in the first of these rounds (together with its ancestors) is considered finalized. At this point, the protocol ensures that it cannot be rolled back.
+
```

Comment:
Nitpicky, I understand what you mean here but; 

1) a block **always** contains a quorum certificate. (The quorum certificate is sort of a 'verified reference'  to the parent block, not the previous round per se. 
The block _may_ contain a timeout certificate, in particular the block contains a timeout certificate if and only if the previous round timed out.

2) The two consecutive blocks must also be within the same epoch.

Reply:
I don't know the context of this paragraph, but here we are defining the new consensus protocol in terms of the new one, does the reader know of slots at this point when reading through the documentation?

Reply:
No, they would not know it in this topic because it is a glossary, so more like a point of reference from topics to further descriptions. I doubt anyone besides me has read the glossary from top to bottom.